rpmfusion-free-rawhide.repo

```diff
@@ -3,7 +3,8 @@ name=RPM Fusion for Fedora Rawhide - Free
 #baseurl=http://download1.rpmfusion.org/free/fedora/development/$basearch/os/
 metalink=https://mirrors.rpmfusion.org/metalink?repo=free-fedora-rawhide&arch=$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
```

Comment:
Fedora has gpgsigned the rawhide repo only recently, Unfortunately, we aren't ready for this yet.
We would need to implement autosigner which requires fedmsg which requires freeipa.

We plan to do this, but not the dirty hacky way. Once this is implemented, the infra will be completely autonomous.
You are welcomed to help from the infra side !